### DISCUSSION TITLE
Resource Schema - shall we add the passwords even when auto-generated by the lz file

### DISCUSSION THREAD

**Discussion by mosharafMS** [Azure/CanadaPubSecALZ#258] [Ideas]
In the sqldb schema, the password is not added as a parameter, probably because the landing zone is responsible for generating the password and passing it. 
/schemas/latest/landingzones/types/sqldb.json

Question, shall we keep it as is or add the password because whether the calling file generate it or not, it's a required (in some cases) parameter. 

@SenthuranSivananthan

**Comment by SenthuranSivananthan** [comment]
Since the parameter files are kept in git, we can't store any passwords in them.  The passwords are generated & stored in Key Vault which keeps the Git repo & git history clean.
